ipfshttpclient/client/__init__.py

```diff
@@ -254,19 +260,55 @@ def add_bytes(self, data: bytes, **kwargs):
 		----------
 		data
 			Content to be added as a file
+		trickle
+			Use trickle-dag format (optimized for streaming) when generating
+			the dag; see `the old FAQ <https://github.com/ipfs/faq/issues/218>`_
+			for more information
+		only_hash
+			Only chunk and hash, but do not write to disk
+		chunker
+			The chunking algorithm to use
+		pin
+			Pin this object when adding
+		raw_leaves
+			Use raw blocks for leaf nodes. (experimental). (Default: ``True``
+			when *nocopy* is True, or ``False`` otherwise)
+		cid_version
+			CID version. Default value is provided by IPFS daemon. (experimental)
 
 		Returns
 		-------
 			str
 				Hash of the added IPFS object
 		"""
+
+		opts = {
+			"trickle": trickle,
+			"only-hash": only_hash,
+			"pin": pin,
+			"raw-leaves": raw_leaves if raw_leaves is not None else nocopy,
+		}  # type: ty.Dict[str, ty.Union[str, bool]]
```

Comment:
From Python 3.6 forward, it's preferable to declare the type on the variable rather than in a comment.

Change to `opts: ty.Dict[str, ty.Union[str, bool]] = { ...`

Reply:
Done.